specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1007 @@
+
+   1. For Subscriptions, if delivery of the reply event fails with a retryable
+      error, the entire delivery of the event to MUST be retried up to `retry`
+      times (subject to congestion control), following the `backoffPolicy` and
+      `backoffDelay` parameters if specified.
+
+1. If all retries are exhausted for either the original delivery or the retry,
+   or if a non-retryable error is received, the event MUST be delivered to the
+   `deadLetterSink` in the delivery options. If no `deadLetterSink` is
+   specified, the event is dropped.
+
+   The implementation MAY set additional attributes on the event or wrap the
+   failed event in a "failed delivery" event; this behavior is not (currently)
+   standardized.
+
+   If delivery of the reply event fails with a retryable error, the delivery to
+   the `deadLetterSink` SHOULD be retried up to `retry` times, following the
+   `backoffPolicy` and `backoffDelay` parameters if specified. Alternatively,
+   implementations MAY use an equivalent internal mechanism for delivery (for
+   example, if the `ref` form of `deadLetterSink` points to a compatible
+   implementation).
+
+## Detailed Resources
+
+The following schema defines a set of REQUIRED or RECOMMENDED resource fields on
+the Knative resource types. Whether a field is REQUIRED or RECOMMENDED is
+denoted in the "Schema Requirement" column. Additional `spec` and `status`
+fields MAY be provided by particular implementations, however it is expected
+that most extension will be accomplished via the `metadata.labels` and
+`metadata.annotations` fields, as Knative implementations MAY validate supplied
+resources against these fields and refuse resources which specify unknown
+fields. Knative implementations MUST NOT require `spec` fields outside this
+implementation; to do so would break interoperability between such
+implementations and implementations which implement validation of field names.
+
+### Broker
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource. The `apiVersion` is `eventing.knative.dev/v1` and the `kind` is
+`Broker`.
+
+#### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>config</code></td>
+    <td><a href="#kreference">KReference</a><br/>(OPTIONAL)</td>
+    <td>A reference to an object which describes the configuration options for the Broker (for example, a ConfigMap).</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a><br/>(OPTIONAL)</td>
+    <td>A default delivery options for Triggers which do not specify more-specific options. If a Trigger specifies _any_ delivery options, this field MUST be ignored.</td>
```

Comment:
the \_any\_ shows the underscores since you're not in MD world any more

Reply:
@evankanderson I feel like I've brainwashed you and I love it 😂